### DISCUSSION TITLE
Add Browser to Share Screen

### DISCUSSION THREAD

**Discussion by shadowscott** [UInt2048/CyberKit#13] [Ideas]
Would be a nice feature to "Open in Cyberkit" when you're on a site that comes up unable to load.

**Comment by UInt2048** [comment]
This exists in CyberKit Fennec, which is now released for iOS 13 in v0.0.8 alpha.

**Comment by plia7** [comment]
> This exists in CyberKit Fennec, which is now released for iOS 13 in v0.0.8 alpha.

How do I open a site from iOS 14.4.1 Safari Plus in CyberKit Fennec? I don't see a shortcut to open it with, that's when I do a long press on a link to see a menu. Would you mind providing a screenshot please showing where this menu item exists or instructions how to see it?

Thank you.

**Reply by UInt2048** [reply]
@plia7 It should be in the share sheet menu

**Reply by plia7** [reply]
> @plia7 It should be in the share sheet menu

Ok I do a long press on a link in Safari, I click Share, then I scroll right in the icons and I see the blue CyberKit icon, when I click on it, nothing happens.

Update: Looks like it only works if I open CyberKit app prior to clicking the icon. Is it the right behavior or should it work without me opening CyberKit beforehand? I created a ticket for it: https://github.com/UInt2048/CyberKit/issues/28

**Reply by UInt2048** [reply]
@plia7 Discussion will be continued at #28
